src/daomop/storage.py

```diff
@@ -1348,11 +1345,15 @@ def isfile(uri):
 
 
 def copy(source, destination):
-    """Copy a file to/from VOSpace. With upto 10 retries on errors"""
+    """Copy a file to/from VOSpace. With upto 10 retries on errors,
+
+    :return: content disposition value from data service
+    :rtype: basestring
+    """
     count = 1
     while True:
         try:
-            return vospace.client.copy(source, destination)
+            return vospace.client.copy(source, destination, disposition=True)
```

Comment:
I made a comment here but I think it wasn't on the pull request so I don't know if you'll see it... but disposition isn't a parameter for vospace.client.copy.